recipe_scrapers/moulinex.py

```diff
@@ -0,0 +1,45 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+from ._utils import re
+
+
+class Moulinex(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "moulinex.fr"
+
+    def author(self):
+        return self.schema.author()
+
+    def title(self):
+        return self.schema.title()
+
+    def total_time(self):
+        return self.schema.total_time()
+
+    def yields(self):
+        return self.schema.yields()
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        ingredients = self.schema.ingredients()
+        spaced_ingredients = [
+            re.sub(
+                r"(\d)(?![ .\d])", r"\1 ", ingredient
+            )  # Insert space if number is not followed by a space, period, or another number
```

Comment:
What do you think about this alternative way to express this -- I _think_ it's equivalent, but knowing regexes, there could be some discrepancies that I'm not thinking of:

```suggestion
                r"([0-9])([^ .0-9])", r"\1 \2", ingredient
            )  # Insert space if number is not followed by a space, period, or another number
```

Reply:
Yes that looks equivalent to me !